### PR TITLE
Transfer learning example

### PR DESCRIPTION
**Pull request by abhi-gm**
## **README.md**

Changed 'image' to 'img'  in the return statement, because the image is read and assigned to variable img

## **CIFAR10_Classification_transfer_Learning.ipynb**

Adding this notebook to provide an example about how to use the Microsoft Vision Model for transfer learning with feature extraction

-  This notebook demonstrates the way to do transfer learning with a neural network.
- Transfer learning is demonstrated by taking [1,2048] features from the output of the Vision Model into a custom-built fully connected network.
- Code to slice the features from the Vision Model with the corresponding label to be used transfer learning and for storing.

### CODE REVIEW

**Comment by zlenyk** [comment]
Discussed offline: change optimizer to Adam yields almost 2% test accuracy gain

**Comment by abhi-gm** [comment]
> Discussed offline: change optimizer to Adam yields almost 2% test accuracy gain

Updated the optimizer to Adam
Added dropout layer to prevent overfitting

**Comment by zlenyk** [comment]
Merging, thank you for contribution!
